### DISCUSSION TITLE
Possible Icons Library alternative?

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [CodingGarden/listd#89] [General]
Haven't tested it yet, but seams very flexible.

https://www.npmjs.com/package/unplugin-icons

Hopefully I can test it and leave some feedback.

**Comment by MrAmericanMike** [comment]
I'm testing it today, and we are able to use a wide range of icons collections with this library. For example to continue using tabler we can:

```js
import IconBrandTwitch from "~icons/tabler/brand-twitch";
```

Available Icons set are here: https://icon-sets.iconify.design/

There are details on how to install them all, or select only specific ones in the documentation.

The only thing I don't like is that even when it mentions that auto-import is supported, it's not detailed as to how to get that working with Svelte. And what makes it even worst is that the route for import doesn't exist to it doesn't auto-complete neither.

**Comment by Amaseal** [comment]
I also tried these out for a project recently and unplugin icons seem great!
